### ISSUE TITLE
Feature Request: Add badges

### ISSUE THREAD

**Issue by brenoepics** [brenoepics/vmd-action#4]
this is the current vue-mess-detector output:
```json
  "codeHealthOutput": [
    {
      "info": "Found <bg_err>0 errors</bg_err>, and <bg_warn>30 warnings</bg_warn>, <bg_info>326 lines</bg_info> of code in <bg_info>10 files</bg_info>"
    },
    {
      "info": "Code Health: [<bg_ok>______________________________________________________</bg_ok><bg_warn>______</bg_warn><bg_err></bg_err>] 91%\n"
    },
    {
      "info": "<bg_info>Code health is OK: 91%</bg_info>\n"
    }
  ],
```
Would be cool something like:
```json
  "codeHealthOutput": [
    {
      "errors": 0,
      "warnings": 30,
      "lines": 326,
      "files": 10,
      "code-health": "91%"
    },
  ],
```
So we can have badges and even more customized templates.

Similar to:

https://shields.io/badges

![Static Badge](https://img.shields.io/badge/any_text-you_like-blue)
![Static Badge](https://img.shields.io/badge/100%25-1?style=flat&label=Code%20Health&labelColor=8A2BE2)

**Comment by brenoepics** [comment]
this issue fixes https://github.com/brenoepics/vmd-action/blob/198e2c81c1f6e08b0c55af4e6e8c88ca338da4ff/src/templates/comment.ts#L20-L20

**Comment by rrd108** [comment]
implemented in 0.54.0

New block added to the `json` output `codeHealth`

**Comment by brenoepics** [comment]
done in `0.0.2`
